EpikCord/__init__.py

```diff
@@ -529,8 +530,22 @@ def figure_out_interaction_class():
             if bool(command_exists): # bool(Filter) returns True every time.
                 if interaction.options:
                     for option in interaction.options:
-                        option_values.append(option.get("value"))
-                await command_exists[0].callback(interaction, *option_values)
+                       option_values.append(option.get("value"))
+                total_checks = len(command_exists[0].check)
```

Comment:
Why is it called check? That'd be the name for the decorator?

Reply:
oh some name switches